test/perf_counters_gtest.cc

```diff
@@ -95,6 +95,22 @@ TEST(PerfCountersTest, Read2Counters) {
   EXPECT_GT(values2[1], 0);
 }
 
+TEST(PerfCountersTest, UnsupportedMutipleHardwareCounters) {
```

Comment:
`UnsupportedMultipleHardwareCounters` is misleading, because it seems to indicate we can't collect more than one counter (i.e. more than, for example, `CYCLES`). How about `ReopenExistingCounters` (no need to say it's unsupported).

Could you also add a test for `PerfCountersMeasurement::StopAndGetMeasurement` usage, that's how the benchmark uses the counters, it is representative of the way the problem surfaced, easier to understand, too (you just need 2 calls to Create and 2 StopAndGetMeasurement calls), and also tests that API's return of the bool.

Also - I think it's be easier in both tests if you just used one of the events, e.g. `kGenericPerfEvent1`. Otherwise it's confusing what the problem is: it's not that you track 2 different events; it's that you track the same one twice.

Reply:
unit tests are updated

Call Creat() twice is not enought to trigger the error. 
Both in #1306 or locally, for single events, it is able to create 5 PerfCounter withough wrong return value.
So I create 10 instances in the unit tests.